ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -878,21 +874,21 @@ module.exports = Backbone.AssociatedModel.extend({
       return undefined
     }
 
-    let results = converter.UTMUPStoLL(utmUpsParts)
+    let { lat, lon } = converter.UTMUPStoLL(utmUpsParts)
 
-    if (!this.isLatLonValid(results.lat, results.lon)) {
+    if (!this.isLatLonValid(lat, lon)) {
       return undefined
     }
 
-    results.lon = results.lon % 360
+    lon = lon % 360
```

Comment:
lat lon will always be valid here after line 879. these will result in noop

Reply:
true, I'll move the lat lon check to after these statements

Reply:
Why don't we have the converter take care of doing the mod?

Reply:
We could submit an issue for updating the separate usng.js library, and then going through the release process for that project. However, that may be out of scope for this ticket. Also, having the usng.js library require a valid latitude and longitude values is a legitimate design.